src/rpc/rawtransaction.cpp

```diff
@@ -879,7 +880,9 @@ static RPCHelpMan sendrawtransaction()
 static RPCHelpMan testmempoolaccept()
 {
     return RPCHelpMan{"testmempoolaccept",
-                "\nReturns result of mempool acceptance tests indicating if raw transaction (serialized, hex-encoded) would be accepted by mempool.\n"
+                "\nReturns result of mempool acceptance tests indicating if raw transaction(s) (serialized, hex-encoded) would be accepted by mempool.\n"
+                "\nIf multiple transactions are passed in, they must be sorted in order of dependency and not conflict with each other.\n"
```

Comment:
nit: clarify which order that is (e.g., `if tx B and C are a child of A the order may be [A, B, C] or [A, C, B]`)

Reply:
nit: clarify is duplicate transactions [A, A] are allowed